lib/MusicBrainz/Server/Report/ReleaseRGDifferentName.pm

```diff
@@ -0,0 +1,38 @@
+package MusicBrainz::Server::Report::ReleaseRGDifferentName;
+use Moose;
+
+with 'MusicBrainz::Server::Report::ReleaseReport',
+     'MusicBrainz::Server::Report::ReleaseGroupReport',
+     'MusicBrainz::Server::Report::FilterForEditor::ReleaseID';
+
+sub table { 'release_rg_different_name' }
+sub component_name { 'ReleaseRgDifferentName' }
+
+sub query {
+    "
+        SELECT
+            r.id AS release_id, rg.id AS release_group_id,
+            row_number() OVER (ORDER BY musicbrainz_collate(r.name), musicbrainz_collate(rg.name))
+        FROM
+            release r
+            JOIN release_group rg 
+            ON r.release_group = rg.id
+        WHERE (SELECT COUNT(*) FROM release WHERE release_group = rg.id) = 1
```

Comment:
I'd recommend joining `release_group_meta` here and checking `rgm.release_count = 1` instead of doing the subquery. Although we have an index on `release.release_group`, I ran both variants on pink and using `release_group_meta` was about 3 seconds faster (~8s vs ~5s).